comprehensive-review/apps/parks-dashboard/.eslintrc

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
Should this file be in the code repository? (Not a typescript guy, so I just wonder)